docs/automations/integrations/jira/update-jira/README.md

```diff
@@ -0,0 +1,28 @@
+# Automatically Update Jira Tickets from PRs
+
+Automatically update Jira tickets based on PR status.
+
+[start:example]
+
+!!! info "Conditions (All Must be True):"
+    * The PR contains a reference to a Jira ticket in the title or description
+
+!!! example "Automation Actions:"
```

Comment:
!!! example is in the wrong place. It should wrap the configuration file immediately below this.